app/views/ideas/card/_show.html.erb

```diff
@@ -53,7 +53,7 @@
 
       </ul>
 
-      <div class='pull-right'>
+      <div class='pull-right avatar'>
         <%= render partial: 'shared/avatar', locals: {user: idea.user, avatar_style: 'width:2.25em;top: 0.5em;position: relative;' }, cache: true %>
```

Comment:
@stedmanblake you can just remove the `top 0.5em` here for the same effect (no need to define a new class).